lua/weapons/gmod_tool/stools/wire_adv.lua

```diff
@@ -413,6 +418,62 @@ elseif CLIENT then
 		return self.AutoWiringTypeLookup_t[inputtype]
 	end
 
+	function TOOL:IsWireEntity(ent)
```

Comment:
This function already exists https://github.com/wiremod/wire/blob/master/lua/wire/wireshared.lua#L451

Reply:
Cheers.